### ISSUE TITLE
Encoding problems with UTF-8 in file names

### ISSUE THREAD

**Issue by v-niskala** [FlameWolf/formzilla#4]
Hi! 👋 
      
Firstly, thanks for your work on this project! 🙂

Today I used [patch-package](https://github.com/ds300/patch-package) to patch `formzilla@3.0.4` for the project I'm working on.

I had issues with filenames in UTF-8 not being handled correctly. I found that Busboy uses `latin1` has the default encoding for the params, including filename. The best solution for this would be if formzilla provided a way to override the busboy config.

Here is the diff that solved my problem:

```diff
diff --git a/node_modules/formzilla/index.js b/node_modules/formzilla/index.js
index ec8c0b1..a44baba 100644
--- a/node_modules/formzilla/index.js
+++ b/node_modules/formzilla/index.js
@@ -18,7 +18,7 @@ const formDataParser = async (instance, options) => {
 		const body = {};
 		const props = request.routeSchema?.body?.properties;
 		const parseField = props ? (name, value) => (props[name]?.type === "string" ? value : tryParse(value)) : (name, value) => value;
-		const bus = busboy({ headers: message.headers, limits });
+		const bus = busboy({ headers: message.headers, limits, defParamCharset: "utf8" });
 		bus.on("file", (name, stream, info) => {
 			results.push(storage.process(name, stream, info));
 			body[name] = JSON.stringify(info);
```

<em>This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296).</em>

**Comment by FlameWolf** [comment]
Thank you for reporting this, I will add a fix.

**Comment by FlameWolf** [comment]
Fixed in version 3.1.0. Relevant commit: https://github.com/FlameWolf/formzilla/commit/50314cd56c0273f39ec408e6b19b56749b1600e7.
